CONTRIBUTING.md

```diff
@@ -0,0 +1,212 @@
+# Contributing
```

Comment:
I think that if we are going to have a Contributing.md file in this repo it should address whether we welcome PRs from folks who are not Plaid employees. Right now the file doesn't address it and it feels a bit ambiguous. Historically, we have not really taken PRs from the community on the Link SDKs, except maybe to fix bugs, and I don't want the wording to make it seem like we welcome PRs and then have people get upset because they spent a bunch of time working on a PR to add some cool feature and then we never approve it. Also, I'm not sure about this one since I didn't really check, but some of the Contributing.md files have historically had instructions that required you to be a Plaid employee in order to follow (like publishing to Plaid's npm repo or running tests against Plaid's CI).

I think it's totally fair to say something like "The primary contributors to this SDK are generally Plaid employees, but we do also welcome PRs from the community to fix bugs." or whatever.  (This also kind of gets back to the whole "why I don't want a CoC" issue.)

Reply:
Makes sense. I put the warning "The primary contributors to this SDK are generally Plaid employees, but we do also welcome PRs from the community to fix bugs" at the top of the file :)